src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -140,19 +142,19 @@
       :fx [[:dispatch [:navigate-to :wallet-transaction-progress]]]})))
 
 (defn- transaction-bridge
-  [{:keys [from-address to-address token-id token-address route]}]
-  (let [{:keys [from bridge-name amount-out gas-amount
+  [{:keys [from-address from-chain-id to-address token-id token-address route data eth-transfer?]}]
+  (let [{:keys [bridge-name amount-out gas-amount
                 gas-fees]}                 route
         eip-1559-enabled?                  (:eip-1559-enabled gas-fees)
         {:keys [gas-price max-fee-per-gas-medium
                 max-priority-fee-per-gas]} gas-fees
         transfer-tx                        (cond-> {:From  from-address
                                                     :To    (or token-address to-address)
                                                     :Gas   (money/to-hex gas-amount)
-                                                    :Value amount-out
+                                                    :Value (when eth-transfer? amount-out)
```

Comment:
is `eth-transfer` specific to eth? or should this be `token-transfer`?

Reply:
Yes, it is specific to ETH (or native token), for the rest of tokens we use `erc20-transfer?`